examples/datasets/plot_nutrimouse.py

```diff
@@ -0,0 +1,171 @@
+"""
+=====================================================
+An mvlearn casestudy: the Nutrimouse dataset
+=====================================================
+
+In this tutorial, we show how one may utilize various tools of mvlearn. We
+demonstrate applications to the Nutrimouse dataset from a nutrition study on
+mice. The data measures 40 mice and has two views: expression levels of
+potentially relevant genes and concentrations of certain fatty acids. Each
+mouse has two labels: it's genetic type and diet.
+
+[1] P. Martin, H. Guillou, F. Lasserre, S. Déjean, A. Lan, J-M.
+    Pascussi, M. San Cristobal, P. Legrand, P. Besse, T. Pineau.
+    "Novel aspects of PPARalpha-mediated regulation of lipid and
+    xenobiotic metabolism revealed through a nutrigenomic study."
+    Hepatology, 2007.
```

Comment:
you have a formatting issue:

https://297-206838300-gh.circle-artifacts.com/0/dev/auto_examples/datasets/plot_nutrimouse.html#sphx-glr-auto-examples-datasets-plot-nutrimouse-py